src/lcm/simulate.py

```diff
@@ -693,7 +587,7 @@ def determine_discrete_dense_choice_axes(variable_info):
 
     """
     discrete_dense_choice_vars = variable_info.query(
-        "~is_continuous & is_dense & is_choice",
+        "~is_continuous & is_choice",
```

Comment:
```suggestion
        "is_choice & is_discrete",
```

Same order as above.